CMakeLists.txt

```diff
@@ -40,6 +41,7 @@ option(VELOX_ENABLE_PARSE "Build parser used for unit tests." ON)
 option(VELOX_ENABLE_EXAMPLES
        "Build examples. This will enable VELOX_ENABLE_EXPRESSION automatically."
        ON)
+option(VELOX_ENABLE_BENCHMARKS "Build velox top level benchmarks." OFF)
```

Comment:
shall we unify VELOX_ENABLE_BENCHMARKS and VELOX_BUILD_BENCHMARKS and use
one variable?

Reply:
it looks from the diff that both starts off
and when VELOX_BUILD_BENCHMARKS is turned on then VELOX_ENABLE_BENCHMARKS get turned on.
but we never turn VELOX_ENABLE_BENCHMARKS on while VELOX_BUILD_BENCHMARKS is off.

I mean at least for now it looks like 1.1 mapping.